dialogy/plugins/text/duckling_plugin/__init__.py

```diff
@@ -737,10 +742,17 @@ def select_datetime(
             entities, lambda entity: entity.dim == const.TIME
         )
 
+        reference_time = unix_ts_to_datetime(
+            self.reference_time, timezone=self.timezone
+        )
+        reference_time = reference_time.replace(
+            hour=reference_time.hour, minute=0, second=0, microsecond=0
```

Comment:
`hour=reference_time.hour` isn't that already happening?

Reply:
my bad, changing this.